### ISSUE TITLE
flashing 'TWRP for HTC One M8 All Variants' fails on M8s

### ISSUE THREAD

**Issue by a-einstein** [TeamWin/Team-Win-Recovery-Project#491]
Details can be found here:
http://forum.xda-developers.com/htc-one-m8/help/make-custom-recovery-htc-one-m8s-t3114245

**Comment by CaptainThrowback** [comment]
M8S isn't an officially supported device, but as far as I know that unofficial version works fine. If it doesn't, you should be posting in that thread you linked, not here.

**Comment by a-einstein** [comment]
Well it does not. I have had the same problems as reported there. That was the reason for searching about it on the web.

The message itself seems to indicated that the img is too large. But people on the thread think it is because of being a different phone. I do not know, I am just starting on that matter.

I guess this is exactly the place to report 'issues' to the developers. I really hope they can fix this, or indicate a workaround. Having just started rooting my new phone, I am already stuck without really getting anywhere.

**Comment by CaptainThrowback** [comment]
Again, the device isn't officially supported, so posting here won't help you. That thread is your best bet, and as long as you follow the instructions in the OP there, the recovery should flash fine.

**Comment by a-einstein** [comment]
Well, let's say that "All Variants" was a bit confusing.

I do not see what my 'best bet' is by a thread that just states there is a problem (like I have), but no solution.
Which 'instructions'? Where?  (Would those be any different than what I have already done?)
What is an 'OP'?

Sorry, I am a 'newby', I have just followed instructions I found for M8 everywhere. 

So please clarify what you mean.

**Comment by CaptainThrowback** [comment]
The M8S isn't a variant of the M8 - it's a completely different device. OP = Original or Opening Post, i.e. the first post of the thread. If you look at the thread you linked, which is specifically for the M8S, there are instructions for flashing custom recovery. Again, posting here isn't going to help you, because the M8S isn't officially supported by TeamWin. All future questions should be posted in that thread - those are the people that actually have the device, and as such, they're the ones that will be able to help you. I don't really understand what you're saying about the thread 'stating there's a problem' when it appears to me that the first post in the thread has instructions in it.

**Comment by a-einstein** [comment]
All right, I see what you mean. 

At first I just noticed people reporting problems and the "max download size" message,  which I also had, and wishing for a recovery. I overlooked the start, which presents a solution. I see you were involved in that. I was also a bit overwhelmed by the 51 pages in the thread.

I will check it out better to see if it can apply to my particular appliance. I am still a bit confused about al this talk about RUU's and the very particular versions of it at Lalleman's. But if I understand it well it should simply be flashing the TWRP 2.8.7 present there, and move on to SuperSU.

Thanks.
